applications/GeoMechanicsApplication/tests/cpp_tests/test_scheme_factory.cpp

```diff
@@ -0,0 +1,88 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "testing/testing.h"
+#include "custom_utilities/scheme_factory.hpp"
+#include "spaces/ublas_space.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+using SparseSpaceType = UblasSpace<double, CompressedMatrix, Vector>;
+using LocalSpaceType = UblasSpace<double, Matrix, Vector>;
+
+KRATOS_TEST_CASE_IN_SUITE(CreateScheme_Throws_WhenSchemeTypeIsUndefined, KratosGeoMechanicsFastSuite)
+{
+    const SchemeFactory<SparseSpaceType, LocalSpaceType> factory;
+
+    const auto parameters =
+    R"(
+    {
+        "solution_type" : "Quasi-Static"
+    }
+    )";
+
+    KRATOS_EXPECT_EXCEPTION_IS_THROWN(factory.Create(Parameters{parameters}),
+                                      "scheme_type is not defined, aborting")
```

Comment:
```suggestion
    KRATOS_EXPECT_EXCEPTION_IS_THROWN(SchemeFactory<SparseSpaceType, LocalSpaceType>::Create(Parameters{parameters}),
                                      "scheme_type is not defined, aborting")
```
(And then variable `factory` is no longer needed.)

Reply:
Same answer as before, the macro doesn't seem to accept it

Reply:
Done using your suggestion